STATUS: INLINE SUCCESSFUL
MODIFIED: org.springframework.boot.actuate.autoconfigure.cloudfoundry.servlet.CloudFoundryMvcWebEndpointIntegrationTests.linksToOtherEndpointsWithRestrictedAccess
MODIFIED FILE: /media/psf/Home/Documents/UT-Austin/course/CS393T/project/ODRepair/experiments/_downloads/spring-boot/spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/cloudfoundry/servlet/CloudFoundryMvcWebEndpointIntegrationTests.java
CLEANER: org.springframework.boot.actuate.autoconfigure.cloudfoundry.servlet.TestMockito.test1
CLEANER FILE: /media/psf/Home/Documents/UT-Austin/course/CS393T/project/ODRepair/experiments/_downloads/spring-boot/spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/cloudfoundry/servlet/TestMockito.java
POLLUTER: org.springframework.boot.actuate.autoconfigure.cloudfoundry.servlet.CloudFoundryMvcWebEndpointIntegrationTests.linksToOtherEndpointsForbidden
POLLUTER FILE: /media/psf/Home/Documents/UT-Austin/course/CS393T/project/ODRepair/experiments/_downloads/spring-boot/spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/cloudfoundry/servlet/CloudFoundryMvcWebEndpointIntegrationTests.java
ORIGINAL CLEANER SIZE: 13
NEW CLEANER SIZE: 6
ELAPSED TIME: 1329.268000125885
==========================
@@ -140,0 +140,8 @@
+ {
+     Class c = Class.forName("org.springframework.boot.actuate.autoconfigure.cloudfoundry.servlet.CloudFoundryMvcWebEndpointIntegrationTests");
+     java.lang.reflect.Field f = c.getDeclaredField("tokenValidator");
+     f.setAccessible(true);
+     Object obj = f.get(null);
+     java.lang.reflect.Method m = org.mockito.Mockito.class.getDeclaredMethod("reset", Object[].class);
+     m.invoke(null, new Object[] { new Object[] { obj } });
+ }
